server/immich-openapi-specs.json

```diff
@@ -3758,16 +3758,60 @@
           "profileImagePath"
         ]
       },
-      "AssetFileUploadDto": {
+      "CreateAssetDto": {
         "type": "object",
         "properties": {
+          "assetType": {
+            "enum": [
+              "IMAGE",
+              "VIDEO",
+              "AUDIO",
+              "OTHER"
+            ],
+            "type": "string"
+          },
```

Comment:
Not sure which is better, but this is different than asset type on `AssetResponseDto`

![image](https://user-images.githubusercontent.com/4334196/218133472-efc7385b-00f8-4690-90ae-6cdcbd4e0701.png)
![image](https://user-images.githubusercontent.com/4334196/218133542-2ddc7465-7985-4232-b41b-ecb0e53f8583.png)

It would be nice to keep it consistent. Can we just copy/keep asset type the same between the two DTOs?

Reply:
Ideally I would keep the $ref variant. When changing the `@ApiProperty` on `AssetResponseDto` the enum name changes which causes problems because it has references in code. So that would mean renaming all references or I could look into applying a patch to the OpenAPI generator like you've done for the mobile API.

I tracked down the responsible [template part](https://github.com/OpenAPITools/openapi-generator/blob/a6eb96ea34231e3e3573d3416879fb56e3c78b59/modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache#L182), so I'll have a look at applying a patch.

Reply:
Applying a patch was succesful, so I reverted my changes to the enum type of `CreateAssetDto` to match the `AssetResponseDto` once again.